### PR TITLE
Updates for Android GraphQL API section

### PR DESCRIPTION
**Pull request by jpignata**
*Issue #, if available:* n/a

*Description of changes:* Updates for Android GraphQL API section

- Add Kotlin samples
- Tune samples for readability
- Minor tweaks to prose

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/graphqlapi/fragments/android/mutate-data.md [hunk 1]
```diff
@@ -2,19 +2,35 @@
...
-    Amplify.API.mutate(
-        todo,
-        MutationType.UPDATE,
-        response -> Log.i("ApiQuickStart", "Updated Blog with id: " + response.getData().getId()),
-        apiFailure -> Log.e("ApiQuickStart", apiFailure.getMessage(), apiFailure)
-    );
-}
+Amplify.API.mutate(
+        ModelMutation.update(todo),
+        response -> Log.i("MyAmplifyApp", "Updated Blog with id: " + response.getData().getId()),
```
Comment (lawmicha) [inline docs/lib/graphqlapi/fragments/android/mutate-data.md:15]: `Updated Todo with id`

File: docs/lib/graphqlapi/fragments/android/mutate-data.md [hunk 2]
```diff
@@ -2,19 +2,35 @@
...
+<amplify-block name="Kotlin">
+
+```kotlin
+val todo = Todo.builder()
+        .name("My updated todo")
+        .build()
+
+Amplify.API.mutate(
+        ModelMutation.update(todo),
+        { response -> Log.i("MyAmplifyApp", "Updated Blog with id: " + response.data.id) },
```
Comment (lawmicha) [inline docs/lib/graphqlapi/fragments/android/mutate-data.md:30]: `Updated Todo with id`

File: docs/lib/graphqlapi/fragments/android/mutate-data.md (hunk 1 above)
Comment (jpignata) [reply docs/lib/graphqlapi/fragments/android/mutate-data.md:15]:
```suggestion
        response -> Log.i("MyAmplifyApp", "Updated Todo with id: " + response.getData().getId()),
```

File: docs/lib/graphqlapi/fragments/android/mutate-data.md [hunk 3]
```diff
@@ -2,19 +2,35 @@
...
```
Comment (jpignata) [inline docs/lib/graphqlapi/fragments/android/mutate-data.md:30]:
```suggestion
        { response -> Log.i("MyAmplifyApp", "Updated Todo with id: " + response.data.id) },
```

File: docs/lib/graphqlapi/fragments/native_common/getting-started/common.md [hunk 4]
```diff
@@ -10,7 +11,7 @@ To setup and configure your application with Amplify API to save items in the ba
 
 ## Provision Backend Storage Services
```
Comment (jpignata) [inline docs/lib/graphqlapi/fragments/native_common/getting-started/common.md:12]:
```suggestion
## Configure API
```

**Pull request merged**